server/controllers/RatingsController.js

```diff
@@ -0,0 +1,121 @@
+import jwt from 'jsonwebtoken';
+
+import db from '../database/models';
+
+const { Rating } = db;
+
+/**
+ * @class RatingsController
+ *
+ * @export {class}
+ */
+class RatingsController {
+  /**
+   * @description - Create New Rating
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   *
+   * @memberOf RatingsController
+   *
+   * @returns {object} response JSON Object
+   */
+  static createRating(req, res) {
+    const decoded = jwt.verify(req.headers.authorization, process.env.SECRET_KEY);
+    return Rating
+      .create({
+        rating: req.body.rating,
+        userId: decoded.userId,
+        articleId: req.articleId
+      })
+      .then((newRating) => {
+        res.status(201).send({
+          ratings: {
+            rating: newRating.rating,
+            articleId: newRating.articleId
+          }
+        });
+      });
+  }
+
+  /**
+   * @description - Get All Ratings For a Specific Article
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   *
+   * @memberOf RatingsController
+   *
+   * @returns {object} response JSON Object
+   */
+  static getRatings(req, res) {
+    let averageRating = 0;
+    return Rating
+      .findAll({
```

Comment:
why not just use findAndCountAll here? instead of the way you wrote it inside the then promise

Reply:
That is true, I could use that. Although I am not just doing a find and count, I also want to get the sum of every rating.